cmd/state-installer/cmd.go

```diff
@@ -345,6 +347,11 @@ func postInstallEvents(out output.Outputer, cfg *config.Instance, an analytics.D
 		return locale.WrapError(err, "err_state_exec")
 	}
 
+	ss := subshell.New(cfg)
+	if filepath.Base(ss.Shell()) == bash.Name && runtime.GOOS == "darwin" {
```

Comment:
Please remove the `filepath.Base()` call and just leave it at `ss.Shell() == bash.Name`

Reply:
🤦‍♂️